core/embed/rust/src/ui/event.rs

```diff
@@ -5,6 +5,7 @@ use core::convert::TryInto;
 pub enum PhysicalButton {
     Left,
     Right,
+    Both,
```

Comment:
we're not getting `Both` as a value from the physical layer, so this should not be here